containers-int-strict/src/Map/Int.hs

```diff
@@ -34,10 +34,10 @@ module Map.Int
        , alter
        ) where
 
-import Prelude hiding (lookup, null)
+import Control.DeepSeq (NFData (..))
 import Data.Coerce (coerce)
-
 import qualified Data.IntMap.Strict as M
```

Comment:
According to our style guide, `qualified` imports should be in separate section at the end of imports separated by empty line. This helps readability in general.